changelog/0.40.md

```diff
@@ -80,3 +80,19 @@ Released 2024-08-05
 - [#2222](https://github.com/elastisys/compliantkubernetes-apps/pull/2222) - documentation: docs: remove oidc-users if restoring harbor to new domain [@viktor-f](https://github.com/viktor-f)
 - [#2223](https://github.com/elastisys/compliantkubernetes-apps/pull/2223) - other: bin: Updated kubectl requirement to match kubespray [@Xartos](https://github.com/Xartos)
 - [#2227](https://github.com/elastisys/compliantkubernetes-apps/pull/2227) - documentation: docs: add instructions for restoring harbor between swift and s3 [@viktor-f](https://github.com/viktor-f)
+
+## v0.40.1
+
+Released 2024-09-16
```

Comment:
```suggestion
Released 2024-09-16

> [!note]
> Known issue: Helmfile diff will always show a deletion of `volumeName` from the Persistent Volume Claim, this can be ignored as it will have no effect on apply.
> This will be fixed in a future version.
```